spec/FeatureSpec.js

```diff
@@ -0,0 +1,93 @@
+describe('BowlingApp',function(){
```

Comment:
Overall, great test coverage and well done for using jasmine-jquery. Use variables or helper functions to avoid the fiddly and repetitive references to elements on the DOM.